features/device-config/service/src/main/java/org/opennms/features/deviceconfig/service/impl/DeviceConfigServiceImpl.java

```diff
@@ -271,6 +271,10 @@ public void setServiceMonitorAdaptor(ServiceMonitorAdaptor serviceMonitorAdaptor
         this.serviceMonitorAdaptor = serviceMonitorAdaptor;
     }
 
+    public void setPollerConfig(PollerConfig pollerConfig) {
+        this.pollerConfig = pollerConfig;
+    }
+
     public PollerConfig getPollerConfig() throws IOException {
         if (this.pollerConfig == null) {
             this.pollerConfig = ReadOnlyPollerConfigManager.create();
```

Comment:
If you're changing the code to require setting a poller config anyway, does this even need to exist anymore?

Reply:
This getter is used in many dcb parts, not sure if it needs to be there after changes, however, I left it to be on the safe side.  But you're right, we can change it to a regular getter.